### PR TITLE
chore: update get user profile info

### PR DESCRIPTION
**Pull request by lmquang**
**What does this PR do?**

-   [x] remove unused logic in `get user profile info`
- [x] send traffic request to audit log svc

### CODE REVIEW

File: pkg/entities/user.go [hunk 1]
```diff
@@ -421,12 +421,17 @@ func (e *Entity) GetUserProfile(guildID, userID string) (*response.GetUserProfil
 		return nil, err
 	}
 
-	currentLevel, err := e.repo.ConfigXPLevel.GetNextLevel(gUserXP.TotalXP, false)
-	if err != nil && err != gorm.ErrRecordNotFound {
-		return nil, err
-	}
+	// currentLevel, err := e.repo.ConfigXPLevel.GetNextLevel(gUserXP.TotalXP, false)
```
Comment (trkhoi) [inline pkg/entities/user.go:424]: remove cmt

File: pkg/entities/user.go [hunk 2]
```diff
@@ -436,25 +441,25 @@ func (e *Entity) GetUserProfile(guildID, userID string) (*response.GetUserProfil
 		progress = 1
 	}
 
-	if gUserXP.Guild == nil {
-		if gUserXP.Guild, err = e.repo.DiscordGuilds.GetByID(guildID); err != nil {
-			return nil, err
-		}
-	}
+	// if gUserXP.Guild == nil {
```
Comment (trkhoi) [inline pkg/entities/user.go:444]: bring back

**Review by trkhoi — APPROVED** [review]

File: pkg/entities/user.go (hunk 2 above)
Comment (lmquang) [reply pkg/entities/user.go:444]: a thay ben discord bot k xai cai guild nay, minh xai cai Guild cho nao v @trkhoi ?

Comment (trkhoi) [reply pkg/entities/user.go:444]: oh hiểu chắc là chưa clean up, v chắc ok r a

**Review by anhnh12 — APPROVED** [review]

**Comment by anhnh12** [comment]
safe to remove all comments

**Pull request merged**